src/store/rootReducer.js

```diff
@@ -0,0 +1,8 @@
+/* eslint-disable no-case-declarations */
```

Comment:
- [ ] Instead of disabling `eslint`, please kindly fix the issue since `linter` are there to help you implement best practices.